test/wrb.js

```diff
@@ -359,20 +359,6 @@ contract("WitnetRequestBoard", ([
       const result = await this.WitnetRequestBoard.readResponseResult(queryId, { from: requester })
       expect(result.value.buffer.data).to.be.equal(resultHex)
     })
-    it("fails if data request bytecode has been manipulated", async () => {
-      // Change the bytecode of the request already posted
-      const newDrBytes = web3.utils.fromAscii("This is a different DR")
-      await requestTestHelper.modifyBytecode(newDrBytes)
-
-      // Should revert when reading the request since the bytecode has changed
-      await expectRevert(
-        this.WitnetRequestBoard.readRequestBytecode(queryId, {
-          from: other,
-          gasPrice: 1,
-        }),
-        "bytecode changed after posting"
-      )
-    })
```

Comment:
Why is this being removed?

Reply:
Given a queryId, it doesn't make too much sense to check whether its request contract changed its bytecode after a response was already reported. 

The only place in which this check makes sense would be when calling `WRB.readRequestBytecode(queryId)` before a query gets solved (as if it was already solved, it should revert with "not in posted status"). So:
- this test will be moved to a more appropiate place
- implementation of `readRequestBytecode` needs to be revisited

In other words, I believe it should not matter at all if a request contract modifies its bytecode at any time after being posted to the WRB, as it will be certain that any eventual response will correspond to the bytecode the request had at the moment the query got posted. This consistency check was done in a previous version of `reportResult`, but not any more.